virustotal_service/forms.py

```diff
@@ -3,6 +3,14 @@
 class VirusTotalConfigForm(forms.Form):
     error_css_class = 'error'
     required_css_class = 'required'
+    vt_api_key_private = forms.BooleanField(required=False,
+                                initial=False,
+                                label='Private key?',
+                                help_text="Is the key a private key?")
+    vt_api_pcap = forms.BooleanField(required=False,
```

Comment:
I'm thinking this might be better off as a run-time option instead of a config option. That way a user can selectively choose to pull PCAPs for a given sample or not.